packages/ceramic-cli/src/declarations.d.ts

```diff
@@ -3,3 +3,5 @@ declare module 'uint8arrays' {
     export function fromString(s: string, enc?: string): Uint8Array
     export function concat(bs: Array<Uint8Array>): Uint8Array
 }
+
+export type { IPFSAPI as IPFSApi } from 'ipfs-core/dist/src/components'
```

Comment:
This seems a bit annoying :/

Reply:
they are still working out some issues :| here's a [link](https://github.com/ipfs/js-ipfs/issues/3358)